_shows/21_22/we_need_to_talk_about_mary.md

```diff
@@ -0,0 +1,57 @@
+---
+title: We Need to Talk About Mary
+playwright: Jen Tucker
```

Comment:
We have a Jenny Tucker and a Jen Tucker - are these the same person?

Reply:
From a quick perusal, it does appear that these are the same person

Reply:
@samozzy change instances of 'Jenny' to 'Jen'?